src/components/fields/Percentage/Settings.tsx

```diff
@@ -0,0 +1,70 @@
+import { Box, InputLabel } from "@mui/material";
+import ColorPickerInput from "@src/components/ColorPickerInput";
+import { ISettingsProps } from "@src/components/fields/types";
+
+import { useState } from "react";
+import { Color, toColor } from "react-color-palette";
+
+export default function Settings({ onChange, config }: ISettingsProps) {
+  const defaultColors = {
+    startColor: toColor("hex", "#ED4747"),
+    midColor: toColor("hex", "#F3C900"),
+    endColor: toColor("hex", "#1FAD5F"),
+  };
+
+  const {
+    startColor,
+    midColor,
+    endColor,
+  }: {
+    startColor: Color;
+    endColor: Color;
+    midColor: Color;
+  } = {
+    ...defaultColors,
+    ...config,
+  };
+
+  const [start, setStart] = useState<Color>(startColor);
+  const [mid, setMid] = useState<Color>(midColor);
+  const [end, setEnd] = useState<Color>(endColor);
+  return (
+    <>
+      <InputLabel>
+        Start Color
```

Comment:
I'm pretty bad at naming things. I'd love to change these labels with a more meaningful alternative.